tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -140,7 +140,7 @@ public static int setDateOrder(int dateOrder) {
     TAJO_MASTER_INFO_ADDRESS("tajo.master.info-http.address", "0.0.0.0:26080", Validators.networkAddr()),
 
     // Tajo Rest Service
-    REST_SERVICE_PORT("tajo.rest.service.port", 26880),
+    REST_SERVICE_ADDRESS("tajo.rest.service-http.address", "0.0.0.0:26880", Validators.networkAddr()),
```

Comment:
IMO, ```tajo.rest.service-http.address``` seems to have a duplicate meaning because rest service is mostly based on http. How about ```tajo.rest.service.address```?